cloudnet-modules/cloudnet-bridge/src/main/java/de/dytanic/cloudnet/ext/bridge/velocity/VelocityCloudNetHelper.java

```diff
@@ -90,11 +90,10 @@ public static NetworkConnectionInfo createNetworkConnectionInfo(Player player) {
                 BridgeHelper.createOwnNetworkServiceInfo()
         );
     }
-
-    public static Optional<RegisteredServer> getNextFallback(Player player) {
+    public static Optional<RegisteredServer> getNextFallback(Player player, RegisteredServer registeredServer) {
         return BridgeProxyHelper.getNextFallback(
                 player.getUniqueId(),
-                player.getCurrentServer().map(ServerConnection::getServerInfo).map(ServerInfo::getName).orElse(null),
```

Comment:
Everything in this class should be fine, why did you change this?

Reply:
This actually fixed it for a single Lobby. This was my first try.

I guess I will revert this then. But why is it in the Bungeecord implementation when it shouldn't be here?

Reply:
Because the velocity api returns optionals, which make a way cleaner api usage possible. Bungee is not doing this, so the code is a bit different.

Reply:
Well I changed it because the current server was null but I think with handleConnectionFailed I can just ignore this. Reverting this now.